src/CSnakes.Runtime/PyObjectTypeConverter.List.cs

```diff
@@ -1,68 +1,22 @@
-using CSnakes.Runtime.CPython;
 using CSnakes.Runtime.Python;
 using System.Collections;
 using System.Runtime.InteropServices;
 
 namespace CSnakes.Runtime;
 internal partial class PyObjectTypeConverter
 {
-    private static ICollection ConvertToList(PyObject pyObject, Type destinationType)
+    private static object ConvertToList(PyObject pyObject, Type destinationType)

```

Comment:
Does this have to be `object`, can't we use `IReadOnlyList`?

Reply:
There are only Generic versions of that interface and we don't know `T`, so no